jupyter-covid19/Dockerfile

```diff
@@ -7,15 +7,15 @@ RUN mkdir /home/jovyan/covid19-notebook
 
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/jhu-summary-overview/COVID-19-JHU_data_analysis_04072020.ipynb /home/jovyan/covid19-notebook/
```

Comment:
can we just copy the whole `covid19-tools/master/covid19-notebooks/` folder instead of one by one?

Reply:
it won't work

Reply:
ah right!